bigquery/api/Snippets/QueryWithTimestampParameters.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright(c) 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// [START bigquery_query_params_timestamps]
+
+using System;
+using System.Globalization;
+using Google.Cloud.BigQuery.V2;
+
+
+public class BigQueryQueryWithTimestampParameters
+{
+    public void QueryWithTimestampParameters(
+        string projectId = "your-project-id"
+    )
+    {
+        var timestamp = new DateTime(2016, 12, 7, 8, 0, 0, DateTimeKind.Utc);
+
+        // Note: Standard SQL is required to use query parameters.
+        var query = "SELECT TIMESTAMP_ADD(@ts_value, INTERVAL 1 HOUR);";
```

Comment:
Ohhh, can't you make a query against a table that has a DataTime column and maybe show something like `where date > @ts_value and date < TIMESTAMP_ADD(@ts_value, INTERVAL 1 HOUR)`?
It's just that using BigQuery to add an hour to a DateTime doesn't look like best practices we should be showcasing.

Reply:
:) 

I'm not a BigQuery expert, but it's what they do in their canonical:
https://github.com/googleapis/python-bigquery/blob/dc389498391798235383fbec20e48a23128ddc6a/samples/client_query_w_timestamp_params.py#L27

Reply:
Hmm, leave as is then. I'll try and ask them about changing the sample.